plugin/trino-ranger/pom.xml

```diff
@@ -0,0 +1,425 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.trino</groupId>
+        <artifactId>trino-root</artifactId>
+        <version>454-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>trino-ranger</artifactId>
+
+    <packaging>trino-plugin</packaging>
+    <name>Apache Ranger authorizer for Trino</name>
+    <description>Apache Ranger authorizer for Trino</description>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <commons.collections.version>3.2.2</commons.collections.version>
+        <commons.lang.version>2.6</commons.lang.version>
+        <commons.lang3.version>3.3.2</commons.lang3.version>
+        <elasticsearch.client.version>7.10.2</elasticsearch.client.version>
+        <graalvm.version>22.3.5</graalvm.version>
+        <gson.version>2.11.0</gson.version>
+        <hadoop-shaded-guava.version>1.2.0</hadoop-shaded-guava.version>
+        <httpcomponents.httpclient.version>4.5.13</httpcomponents.httpclient.version>
+        <httpcomponents.httpcore.version>4.4.14</httpcomponents.httpcore.version>
+        <jackson-annotation.version>2.17.1</jackson-annotation.version>
+        <jersey-client.version>1.19.3</jersey-client.version>
+        <jersey-core.version>1.19.3</jersey-core.version>
+        <junit.jupiter.version>5.10.0</junit.jupiter.version>
+        <junit.version>4.13.2</junit.version>
+        <opensearch.client.version>1.3.15</opensearch.client.version>
+        <opentelemetry.version>1.37.0</opentelemetry.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <ranger.version>2.5.0</ranger.version>
+        <slf4j-api.version>2.0.16</slf4j-api.version>
+        <zookeeper.version>3.5.5</zookeeper.version>
```

Comment:
I do believe that we should be using versions from the main Trino pom.xml, that way we will be sure that these libraries get updated frequently.

Reply:
These libraries are used only within Ranger plugin. Do you prefer to move these version properties to main pom.xml?